src/query/service/src/interpreters/hook/vacuum_hook.rs

```diff
@@ -65,3 +66,7 @@ pub fn hook_vacuum_temp_files(query_ctx: &Arc<QueryContext>) -> Result<()> {
 
     Ok(())
 }
+
+pub fn hook_disk_temp_dir(query_ctx: &Arc<QueryContext>) -> Result<()> {
+    TempDirManager::instance().drop_disk_spill_dir(&query_ctx.get_id())
```

Comment:
We need vacuum to be able to clean up these temporary files when the query is unexpectedly interrupted.